File: pubnub-gson/src/main/java/com/pubnub/api/endpoints/presence/WhereNow.java
package com.pubnub.api.endpoints.presence;

import com.pubnub.api.endpoints.Endpoint;
import com.pubnub.internal.models.consumer.presence.PNWhereNowResult;

public interface WhereNow extends Endpoint<PNWhereNowResult> {
    WhereNow uuid(String uuid);
}

File: pubnub-kotlin/pubnub-kotlin-api/src/main/kotlin/com/pubnub/api/endpoints/presence/WhereNow.kt
package com.pubnub.api.endpoints.presence

import com.pubnub.api.Endpoint
import com.pubnub.api.models.consumer.presence.PNWhereNowResult

/**
 * @see [PubNub.whereNow]
 */
interface WhereNow : Endpoint<PNWhereNowResult> {
    val uuid: String
}
